src/bin/vip-import-sql.js

```diff
@@ -192,13 +214,154 @@ const examples = [
 	},
 ];
 
+const promptToContinue = async ( {
+	launched,
+	formattedEnvironment,
+	track,
+	domain,
+} ): Promise<void> => {
+	console.log();
+	const promptToMatch = domain.toUpperCase();
+	const promptResponse = await prompt( {
+		type: 'input',
+		name: 'confirmedDomain',
+		message: `You are about to import the above tables into a ${
+			launched ? 'launched' : 'un-launched'
+		} ${ formattedEnvironment } site ${ chalk.yellow( domain ) }.\nType '${ chalk.yellow(
+			promptToMatch
+		) }' (without the quotes) to continue:\n`,
+	} );
+
+	if ( promptResponse.confirmedDomain !== promptToMatch ) {
+		await track( 'import_sql_unexpected_tables' );
+		exit.withError( 'The input did not match the expected environment label. Import aborted.' );
+	}
+};
+
+export type validateAndGetTableNamesInputType = {
+	skipValidate: boolean,
+	appId: number,
+	envId: number,
+	fileNameToUpload: string,
+};
+
+export async function validateAndGetTableNames( {
+	skipValidate,
+	appId,
+	envId,
+	fileNameToUpload,
+}: validateAndGetTableNamesInputType ): Promise<Array<string>> {
+	const validations = [ staticSqlValidations, siteTypeValidations ];
+	let tableNamesInSqlFile = [];
```

Comment:
Super minor, but we're not really reassigning this, so I'd drop the variable and just return the appropriate values below:

```suggestion
```